sympy/logic/boolalg.py

```diff
@@ -186,6 +197,10 @@ class BooleanFalse(with_metaclass(Singleton, BooleanAtom)):
     False they act bitwise on 0. Functions in the logic module will return this
     class when they evaluate to false.
 
+    Notes
+    ======
+    See note in sympy.logic.boolalg.BooleanTrue
```

Comment:
I think there may be something you need to add here to make it hot link in Sphinx.